test/fixtures/mix_cut_test/overlayed_cut_manifest.yml

```diff
@@ -0,0 +1,57 @@
+- id: mixed-cut-id
```

Comment:
Note that the `CutSet` is currently a heterogeneous data structure, as `Cut` and `MixedCut` don't have 100% identical interfaces. I'm not sure at the moment whether we should have a separate `MixedCutSet` to avoid this or not.

Reply:
The `Cut` and `MixedCut` semantics are actually a bit different - they do share the ability to load features, provide supervision, the duration attribute - but for `MixedCut` it doesn't make sense to specify e.g. the start or the channel, like for `Cut`.